app/views/layouts/application.html.erb

```diff
@@ -17,8 +17,9 @@
       <p class="alert"><%= alert %></p>
     </main>
     <div class="main-container">
+      <%= render "layouts/header" %>
```

Comment:
move your partial out of the main container
```suggestion
    <%= render "layouts/header" %>
    <div class="main-container">
```

Reply:
done! thanks!